### PR TITLE
Feature editar mi perfil 159956565

### PR DESCRIPTION
**Pull request by manuel-saravia**
Just review from commit 9554d43, because airbnb rules don't allow jsx code in .js files, so i had to delete what I'd done and add it all over again.

### CODE REVIEW

File: src/components/EditProfile.jsx [hunk 1]
```diff
@@ -0,0 +1,579 @@
...
+        case 'nickname':
+          error = 'Ajústese al formato especificado.';
+          break;
+        case 'image':
+          error = 'Debe ser una url válida.';
+          break;
+        case 'document':
+          if (newValue.length < 8) {
+            error = 'Debe ingresar todos los números, incluído el dígito verificador.';
+          } else {
```
Comment (rocespedes) [inline src/components/EditProfile.jsx]: maybe "else" sentence could be in other line

File: src/components/EditProfile.jsx [hunk 2]
```diff
@@ -0,0 +1,579 @@
...
+    });
+  }
+
+  handleSubmit(event) {
+    const { errors } = this.state;
+    const { fields } = this.state;
+    const { match } = this.props;
+    const { userId } = match.params;
+    // const userId = '1'; //for testing
+
```
Comment (rocespedes) [inline src/components/EditProfile.jsx]: unnecessary extra line

File: src/components/EditProfile.jsx [hunk 3]
```diff
@@ -0,0 +1,579 @@
...
+                type: 'url',
+                key: 'image',
+                validPattern: 'https?://.+',
+              }, '200')}
+            </form>
+            <button type="button" className="btn btn-secondary modal-button" onClick={this.closeModal}>Cerrar</button>
+          </div>
+        </Modal>
+
+
```
Comment (rocespedes) [inline src/components/EditProfile.jsx]: unnecessary extra line

File: src/components/EditProfile.jsx [hunk 4]
```diff
@@ -0,0 +1,579 @@
...
+    const { image } = fields;
+    const { hasError } = this.state;
+
+    if (hasError) {
+      return <p>Ha ocurrido un error. Disculpe las molestias.</p>;
+    }
+
+    return (
+      <div className="EditProfile">
+
```
Comment (rocespedes) [inline src/components/EditProfile.jsx]: unnecessary extra line

**Comment by manuel-saravia** [comment]
@rocespedes just made those changes you requested

**Pull request merged**